harvester_e2e_tests/apis/test_images.py

```diff
@@ -161,3 +207,29 @@ def test_create_with_invalid_url(api_client, unique_name):
     assert "no such host" in image_conds[0].get("message")
 
     api_client.images.delete(unique_name)
+
+
+def create_image_url(api_client, display_name, image_url, wait_timeout):
```

Comment:
do we provide `display_name` and `image_url` as a fixture?

and for a test, function name should starts with `test`